admin/orders.php

```diff
@@ -847,11 +856,12 @@ function couponpopupWindow(url) {
                 </td>
                 <?php
                     if ($show_orders_weights === true) {
-                        $products_weight = $order->products[$i]['products_weight'];
-                        if ($products_weight === null) {
+                        $products_weight_unit = $order->products[$i]['products_weight'];
+                        if ($products_weight_unit === null) {
                             $products_weight = '&mdash;';
                         } else {
-                            $products_weight .= ' ' . ltrim(TEXT_PRODUCT_WEIGHT_UNIT, ' ');
+                            $products_weight_total = rtrim(number_format((float)($products_weight_unit * $order->products[$i]['qty']), 4), '0.');
```

Comment:
As opposed to `round()`, is `number_format()` intentionally used so as to apply American formatting rules? (The absence of 3rd, 4th params applies comma and period as separators).

Reply:
`number_format` was chosen so that a value like `20.120` could be trimmed to `20.12` or a value like `20.000` could be trimmed to `20`.  I wasn't aware that formatting rules applied to weights, since that's not dealt with anywhere else in the code when handling weights.

Reply:
Never mind, duh.  I'm (currently) using the default parameters like @drbyte indicated in his initial comment.